### DISCUSSION TITLE
Not getting the option of "Remix in spoke" or "Edit in Spoke"

### DISCUSSION THREAD

**Discussion by Kanishk-Malick** [mozilla/hubs#3122] [Spoke Questions] [answered]
I am Not getting the option of "Remix in spoke" or "Edit in Spoke" while using a scene in Hubs. 
Why ?

**Comment by davegoopot** [comment, marked as answer]
That's probably because the original creator didn't press the option to allow remixing.

Try this one: https://hubs.mozilla.com/scenes/Y89X6aJ
